CSE115A/rideshare-project
# Full Implementation of getGeo & getPrices Functions

Problem
=======
As a developer, 
I want to have all firebase functions connected
So I can fully implement prices output

[Link to Ticket](https://trello.com/c/OknXK9Us)

Solution
========
Connected the functions with ENV variables. GetGeo gets called when we choose a location. 

Steps to Verify:
----------------
1. Checkout this branch and start a local dev server with the appropriate env variables
1. Spin up a functions server as well 
1. Verify that when choosing two locations -- it outputs the prices 

Screenshots:
-----------------------
